index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <meta charset="utf-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+   <!-- End Meta tags  -->
+
+   <!-- Title  -->
+   <title>Software Developer Portfolio | Nwachan Theophile</title>
+   <!-- End Title  -->
+
+   <!-- Styles  -->
+   <link rel="stylesheet" href="style.css">
+
+   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+   <!-- Styles End  -->

```

Comment:
- [ ] [OPTIONAL] Please, consider deleting comments to make your code more cleaner 👍